src/main/java/core/basesyntax/models/OperationDataResult.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.models;
+
+import core.basesyntax.enums.OperationType;
+
+public class OperationDataResult {
+    private int supply;
+    private int buy;
+
+    public OperationDataResult(Operation[] operationData) {
+        this.supply = getOperationSum(operationData, OperationType.SUPPLY.name().toLowerCase());
+        this.buy = getOperationSum(operationData, OperationType.BUY.name().toLowerCase());
+    }
+
+    @Override
+    public String toString() {
+        return "supply," + supply + System.lineSeparator()
+                + "buy," + buy + System.lineSeparator()
+                + "result," + getResult();
```

Comment:
Save strings to constants and better use StringBuilder for concatenation of strings